code/modules/admin/verbs/randomverbs.dm

```diff
@@ -513,6 +513,81 @@ Traitors and the like can also be revived with the previous role mostly intact.
 
 	return new_character
 
+/client/proc/spawn_character_mob()
+	set category = "Special Verbs"
+	set name = "Spawn Character As Mob"
+	set desc = "Spawn a specified ckey as a chosen mob."
+	if(!holder)
+		src << "Only administrators may use this command."

```

Comment:
Its better to use to_chats instead of << for sending messages, both here and in few cases below

Reply:
Not really needed but I'll do it just for you, precious.